packages/jsii-rosetta/lib/rosetta-reader.ts

```diff
@@ -291,7 +290,7 @@ export class RosettaTabletReader {
   }
 
   public get hasErrors() {
-    return this.diagnostics.some(isError);
+    return this.diagnostics.some((d) => d.isError);
```

Comment:
This is a pesky little bug independent of this PR that came about when the type of `this.diagnostics` was changed. It means that assemblies in `strict` mode would never actually bail (in my case, tests in `strict` mode would succeed) since `rosetta.hasErrors()` would always return `false`.